cli/aws_orbit/remote_files/kubectl.py

```diff
@@ -623,6 +623,13 @@ def deploy_env(context: "Context") -> None:
         sh.run(f'kubectl patch deployment -n kubeflow jupyter-web-app-deployment --patch "{patch}"')
         sh.run("kubectl rollout restart deployment jupyter-web-app-deployment -n kubeflow")
 
+        _apply_deployment_patch_force_env_nodes("istio-system")
+        _apply_deployment_patch_force_env_nodes("knative-serving")
+        _apply_deployment_patch_force_env_nodes("cert-manager")
```

Comment:
cert-manager shouldn't need patching. we manage this with kustomizations. patching it will push it to ec2 in an isolated env. instead we should make sure the overlays for isolated/non-isolated set the node-selector appropriatel

Reply:
You are correct